hello-world/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# syntax=docker/dockerfile:1
+FROM stackable/image/java-base
+
+ARG PRODUCT
+ARG RELEASE
+
+LABEL name="Hello World" \
+      maintainer="info@stackable.de" \
+      vendor="Stackable GmbH" \
+      version="${PRODUCT}" \
+      release="${RELEASE}" \
+      summary="The Stackable image for Apache Druid." \
+      description="This image is deployed by the Stackable Operator for Apache Druid."
+
+# https://github.com/hadolint/hadolint/wiki/DL4006
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+RUN microdnf update && \
+    microdnf clean all
+
+USER stackable
+WORKDIR /stackable
+
+RUN curl -L https://repo.stackable.tech/repository/packages/hello-world/hello-world-${PRODUCT}.jar -o hello.jar
+
+CMD ["java", "-jar", "hello.jar", "--spring.config.location=file:mount/config/"]
```

Comment:
Any particular reason to use `file:mount/config/` and not `file:/stackable/config/`? I *think* this aligns better with our current operators

Reply:
Ah! Good thing you bring it up, because I got this from the Hive operator and there the configmap is mounted under /stackable/mount/config.

I was also slightly confused initially, and not quite sure why it is like that, I'm happy to switch it, but should I investigate further?

Reply:
https://github.com/stackabletech/hive-operator/blob/main/rust/crd/src/lib.rs#L32

Reply:
```
pub const STACKABLE_CONFIG_DIR: &str = "/stackable/config";
[...]
pub const STACKABLE_CONFIG_MOUNT_DIR: &str = "/stackable/mount/config";
```
=> We need to mount the ConfigMap (here at  "/stackable/mount/config"). As the ConfigMap is mounted readonly it the files can not be changed (and also should not). We have to run sed on the files, as we need to put in some secrets.
So we copy whatever is mounted to "/stackable/config", so that we can run sed on it and have the final config in "/stackable/mount/config". I don't think we need the complexity here, the important bit for me is that the **effective** config is below /stackable/config (though not all operators do so :see_no_evil: )

Reply:
Good point. I'll change it!